ironfish/src/rpc/routes/wallet/types.ts

```diff
@@ -80,6 +68,51 @@ export const RpcWalletNoteSchema: yup.ObjectSchema<RpcWalletNote> = yup
   })
   .defined()
 
+export type RpcAccountTransaction = {
+  hash: string
+  fee: string
+  blockHash?: string
+  blockSequence?: number
+
+  notesCount: number
+  spendsCount: number
+  mintsCount: number
+  burnsCount: number
+  expiration: number
+  timestamp: number
+  submittedSequence: number
+  confirmations: number
```

Comment:
nit - I think it's a little odd that we return confirmations as part of the response. Maybe we should deprecate this field?